app/appsettings.prod.json

```diff
@@ -1,3 +1,4 @@
 {
-    "baseUrl": "http://bolaget-beta.cacti.se"
+    "inventoryApiBaseUri": "http://bolaget-beta.cacti.se",
```

Comment:
Might as well change this to the HTTPS enabled test domain while we're at it; https://api.daata.xyz

Reply:
I'll pop it on the trello board